zshrc.zsh

```diff
@@ -101,6 +101,7 @@ path=(
   $HOME/code/dotfiles-personal/bin
   $HOME/code/dotfiles/bin
   $HOME/bin/crystal-symlinks
+  $HOME/bin
```

Comment:
I downloaded the `mitmproxy` and other binaries to this directory. That was easier than downloading them to `~/.local/bin`.